Gaijinizer patch/Enemies.txt

```diff
@@ -635,11 +631,11 @@ intention of the other's struggle when bound.#MTLed
 また、その独特の肌触りゆえに、自ら足先に
 ひざまずく男性犠牲者も少なくはない。
 > CONTEXT: Enemies/508/BookDescription/3
-The ankles on Lampas's ankles are a collection of lightly
-fibers
-It can efficiently absorb the milking sperm in the foot.
-In addition, because of its unique touch, in his own foot
-There are also many male victims who are kneeling.#MTLed
+The "anthers" on a lampas' ankles are a collection of
+soft fibres that can efficiently absorb semen ejaculated
+on her feet. In addition, because of the unique
+sensation of her skin, there are few male victims able
+to get back to their feet.#RonIffy
```

Comment:
Let's avoid two repeating mentions of the word feet here and clean up that last sentence:
>_**Because of the unique sensation of her skin, there are few male victims
able to get back up once they've fallen.**_

(There's something about men willing kneeling, but I don't know enough about the translation here for that...)

Reply:
Now that I look at it again, it might actually be saying something along the lines that there are plenty of men willing to kneel at her feet because of their unique sensation.

Reply:
Reworded but the original sentence is really awkward. The sentence basically says something along the lines of "there are not not many men who would fall to their knees before her."